editor/editor_properties.cpp

```diff
@@ -3064,7 +3086,11 @@ void EditorPropertyNodePath::_node_assign() {
 }
 
 void EditorPropertyNodePath::_node_clear() {
-	emit_changed(get_edited_property(), NodePath());
+	if (pointer_mode) {
+		emit_changed(_get_meta_pointer_property(), (Object *)nullptr);
```

Comment:
Same here. Since it's clearing it still works like this, but it should be consistent IMO.